views/js/qtiCreator/widgets/interactions/selectPointInteraction/states/Map.js

```diff
@@ -196,7 +196,14 @@ define([
                 } else {
                     //move the score back the shape and show the popup
                     shape.toFront();
-                    $('#score-popup-' + shape.id).show(); 
+                    $('#score-popup-' + shape.id).show();
+
+                    //the click on the cross hides the popup
+                    $('.mapping-editor').on('click', '.closer', function(){
+                        $('#score-popup-' + shape.id).hide();
```

Comment:
You should keep a ref to the element instead of querying the DOM multiple times : 
```
var $scorePopup = $('#score-popup-' + shape.id)
```